load/S3Utility.py

```diff
@@ -48,10 +48,10 @@ def uploadAllFilesets(self, filesets):
 						print("uploadAllFilesets. outFileset NOT added to database list: %s" % (outFileset.csvFilename))
 			elif inp.typeCode == "text":
 				subTypeCode = inp.subTypeCode()
-				if subTypeCode in {"text_plain", "text_usx", "text_json"}:
-					InputFileset.database.append(inp) ## Temp until text_format is ready
+				if subTypeCode in {"text_plain"}:
```

Comment:
@bradflood, my bad, I changed it again because if the fileset is text_usx or text_json, the uploadFileset method will add the fileset to the InputFileset.database slice and upload the file. The issue was that it was not invoking the method uploadFileset for text_usx and text_json.

Reply:
what was the reason you changed it initially? You just undid it, so I would like to know the effect on this on the original issue you were addressing